Appveyor fails on native Windows with minimum package level

The latest master version fails on Appveyor on all variants with native Windows and package level minimum. See the list of runs of the latest manual-ci-run branch: https://ci.appveyor.com/project/KSchopmeyer/pywbemtools/builds/26788198

It always seems to be the same issue, namely that `assert_rc()` fails with "AssertionError: Unexpected exit code (expected 0, got 1)".

Note that the same Python versions on native Windows succeed with the latest package level, so it must have to do something with the level of dependent packages.

--- a/dev-requirements.txt
+++ b/dev-requirements.txt
@@ -10,7 +10,9 @@
 # Direct dependencies:
 
 # Unit test (imports into testcases):
-pytest>=3.3.0
+# pytest 5.0.0 requires Python version 3.5
+pytest>=3.3.0,<5.0.0; python_version <= '3.4'
+pytest>=3.3.0; python_version >= '3.5'
 funcsigs>=1.0.2; python_version == '2'
 
 # Note: python-coveralls 2.9.1 has requirement coverage==4.0.3, and therefore

--- a/minimum-constraints.txt
+++ b/minimum-constraints.txt
@@ -53,7 +53,8 @@ click-spinner==0.1.8
 click-repl==0.1.6
 asciitree==0.3.3
 tabulate==0.8.2
-prompt-toolkit==2.0.1
+prompt-toolkit==1.0.15; python_version == '2.7'
+prompt-toolkit==2.0.1; python_version >= '3.4'
 
 # Indirect dependencies for install (not in requirements.txt)
 

